pkg/roll/run_update.go

```diff
@@ -363,7 +363,7 @@ func (u *update) countHealthy(id rcf.ID, checks map[string]health.Result) (rcNod
 
 	ret.Desired = rcFields.ReplicasDesired
 
-	currentPods, err := rc.New(rcFields, u.kps, u.rcs, u.sched, u.labeler, u.logger).CurrentPods()
+	currentPods, err := rc.New(rcFields, u.kps, u.rcs, u.sched, u.labeler, u.logger, nil).CurrentPods()
```

Comment:
sure, although I don't anticipate `CurrentPods()` alerting ever. the roll farm may alert in other places though so I'll do it.